### DISCUSSION TITLE
Is it possible to make the process of taking notes structured and presentable?

### DISCUSSION THREAD

**Discussion by svnny314** [windingwind/zotero-better-notes#845] [Ideas]
Can we take notes on the right side like marginnote? Make a structured presentation of the notes being taken, or have a table of contents next to it to quickly locate them, making it easy to adjust the position and logic between notes at any time？

**Comment by windingwind** [comment]
sorry i don't understand.

**Reply by svnny314** [reply]
![微信图片_20231229154544](https://github.com/windingwind/zotero-better-notes/assets/154224904/1717971f-e24f-4de1-a7f4-e95a24db43a0)
![微信图片_20231229154554](https://github.com/windingwind/zotero-better-notes/assets/154224904/206adbc8-7bbd-43df-a4ca-2b13d05830ca)
em......I mean , when we are making notes on the right side, could it be displayed like this. That way we can see at a glance the primary, secondary and tertiary headings and the logical relationship between them. 。。。。 (sorry，my poor English。。。)

**Reply by windingwind** [reply]
Sorry but no, that's not what this plugin is designed for.

**Reply by svnny314** [reply]
fine, thanks.
